src/node.js

```diff
@@ -388,6 +396,53 @@
       nextTickQueue.push(obj);
       tickInfo[kLength]++;
     }
+
+    // Internal callbacks that must be run when there is no more micro-tasks
+    // scheduled for this turn.
+    function lateTick(callback) {
+      lateTickQueue.push(callback);
+      scheduleMicrotasks();
```

Comment:
> In fact you can comment out the line and not a single test will pass anymore.

it means that something is terribly wrong in some other place. `scheduleMicrotasks` is supposed to be called only from nextTick queue processing code